README.md

```diff
@@ -27,43 +27,13 @@ Prior knowledge of Angular is recommended.
 
 ## Getting Started
 
-For an existing Angular application,
+For an existing Angular CLI application,
 
-1. **Install Fundamental-ngx and Fundamental Styles.**
+1. **Install Fundamental-NGX.**
 
-    ```
-    npm install --save fundamental-styles fundamental-ngx
-    ```
-
-2. **Include Fundamental Styles CSS in the `styles` array of the `angular.json` file.**
-
-    ```
-    "node_modules/fundamental-styles/dist/fundamental-styles.min.css"
-    ```
-
-    _Note the path may be different if your CLI configuration is not in the root of your project directory._
-
-3. **Configure your animations.**
-
-    Install angular animations by running the following.
-    ```
-    npm install @angular/animations
-    ```
-    
-    Once the above package is installed, import `BrowserAnimationsModule` to enable the animations.
-    
-    ```typescript
-    import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
-    
-    @NgModule({
-      ...
-      imports: [BrowserAnimationsModule],
-      ...
-    })
-    export class DemoModule { }
-    ```
+    `ng add fundamental-ngx`
     
-    Alternatively, you can disable the animations by importing `NoopAnimationsModule` instead.
+    *If you do not use the Angular CLI or if this command does not work for you, please see the [full installation guide](https://github.com/SAP/fundamental-ngx/wiki/Full-Installation-Guide-%5B0.9.0%5D).*
```

Comment:
I would rename the wiki(link) without having the version as part of it.
Maybe we can separate the content before `0.9` and after

Reply:
Yeah it's just there to not confuse people. It will be removed after 0.9.0 is released.

Reply:
then we need to update `README.md` again

Reply:
Okay let's confuse people 😀